documentation/manual/src/main/asciidoc/modules/query.asciidoc

```diff
@@ -8,6 +8,7 @@ With Hibernate OGM, you have a few alternatives that should get you covered:
 * Use JPQL - only for simple queries for now
 * Use the NoSQL native query mapping the result as managed entities
 * Use Hibernate Search queries - primarily full-text queries
+* Use the stored procedure mapping the result as managed entities
```

Comment:
Maybe `Use stored procedures mapping the results to managed entities` would be better?